pinot-common/src/main/java/com/linkedin/pinot/common/utils/FileUploadUtils.java

```diff
@@ -119,6 +121,9 @@ public InputStream createInputStream() throws IOException {
         }
       })};
       method.setRequestEntity(new MultipartRequestEntity(parts, new HttpMethodParams()));
+      Header header = new Header("If-Match", crc);
```

Comment:
Can u get the string "If-Match" from some apache class?

Reply:
None of the apache classes have the if-match header